### ISSUE TITLE
Foreman unable to render site after hdm plugin installation

### ISSUE THREAD

**Issue by tuxmea** [betadots/foreman_hdm#23]
Adding foreman hdm plugin causes foreman to not be able to start properly.

Logfile shows error:

```
ActionView::Template::Error (Can't find entry point 'foreman_hdm:global' in webpack manifest):
 1307acc8 |     13:     <%= stylesheet_link_tag *webpack_asset_paths('foreman-vendor', :extension => 'css') %>
 1307acc8 |     14:     <%= stylesheet_link_tag *webpack_asset_paths('bundle', :extension => 'css') %>
 1307acc8 |     15:     <%= stylesheet_link_tag 'application' %>
 1307acc8 |     16:     <%= webpacked_plugins_with_global_css %>
 1307acc8 |     17:     <%= yield(:stylesheets) %>
 1307acc8 |     18:
 1307acc8 |     19:     <%= csrf_meta_tags %>
 1307acc8 |
```

**Comment by oneiros** [comment]
Sorry, it has been a long time since I worked on this and my memory is kind of fuzzy when it comes to details.

That being said, I think two things could be important to note here:

1. The most important question is _how_ you installed it. The only way I was able to run it, was with a local checkout of foreman. The documentation I found showing how to install a plugin in development as a ruby gem into a running (production like) foreman installation did _not_ work and failed with a very similar error. In that case the installation process does not build the webpack assets. IIRC at the time there was no way for the plugin to build the assets itself, webpack had be run from the foreman directory in order for everything to be build correctly.

    We discussed this back then and after some back and forth with foreman devs the info we got was, that we need to build an rpm package for the plugin and then the installation process will somehow take care of building the assets correctly.

2. Which version of foreman are you using? As I said, it has been a while and in the meantime, foreman has probably released a couple of new versions with possible changes.
